ACM_General/home/templates/home/base.html

```diff
@@ -40,26 +43,25 @@
 
 {% block body_header %}
   <header>
-    <div class="constraint-wrapper">
-      <div class="header-acm-seal-wrapper">
-        <div class="header-acm-seal">
+  <div class="constraint-wrapper">
+      <div class="header__logo-wrapper">
+        <div class="header__logo">
           <img src="{% static "/home/img/core/acm.png" %}">
           <h3>Association for<br>Computing Machinery</h3>
         </div>
       </div>
 
-      <div class="header-welcome-wrapper">
-        <h1 class="header-welcome">We are Missouri S&T - ACM</h1>
+      <div class="header__welcome-wrapper">
+        <h1 class="header__welcome">We are Missouri S&T - ACM</h1>
       </div>
     </div>
   </header>
 {% endblock %}
 {% block body_navigation %}
-  <div class="top-bar"></div> <!-- mobile menu bar -->
   <div id="menu-wrapper"><!-- mobile menu -->
     <input type="checkbox" id="menu" name="menu" class="menu-checkbox">
     <div class="menu">
-      <label class="menu-toggle" for="menu"><span>Toggle</span></label>
+      <label class="menu-toggle" for="menu"><span></span><span></span><span></span></label>
```

Comment:
A lot of empty spans here? I'm assuming the is hamburger formatting or something of the like